test/common/chromium-cdp-driver.js

```diff
@@ -241,10 +241,10 @@ function getFrameExecId(frame) {
         frame = "cockpit1";
     var frameId = frameNameToFrameId[frame];
     if (!frameId)
-        throw Error(`Frame ${frame} is unknown`);
+        return -1;
```

Comment:
Needs the same thing on firefox-cdp-driver.js? Or more cleanly, leave this alone and instead catch the exception in the wait loop only, but not anywhere else?